pkgs/applications/networking/instant-messengers/telegram/telega-server/default.nix

```diff
@@ -0,0 +1,31 @@
+{ fetchFromGitHub, stdenv, tdlib }:
+
+stdenv.mkDerivation rec {
+  version = "2019-07-10";
```

Comment:
Any reason to not use the v0.2.5 release?

Reply:
The latest `v0.2.5` is far behind (like 250 commits). I'm still pretty new to Nix so I don't know if it's ok to use such versions.

Reply:
If there is a specific functionality missing it's generally okay to use an unstable version, but we want to ask upstream to make an official release (just open a ticket with their github and link it there).

Reply:
Gotcha, opened an issue https://github.com/zevlg/telega.el/issues/66

Reply:
https://github.com/zevlg/telega.el/issues/66#issuecomment-513186551

Reply:
Updated version to `0.4.0`